### PR TITLE
Process existing operations on start

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Implemented processing of existing **in progress** operations on start

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/storage/dbsession/read.go [hunk 1]
```diff
@@ -43,6 +44,26 @@ func (r readSession) GetOperationByID(opID string) (OperationDTO, dberr.Error) {
...
+	var operations []OperationDTO
+
+	_, err := r.session.
+		Select("*").
+		From(postsql.OperationTableName).
+		Where(stateCondition).
+		Where(typeCondition).
+		Load(&operations)
+	if err != nil {
+		if err == dbr.ErrNotFound {
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/storage/dbsession/read.go]: if there is no any in progress operations, it is better to return empty list. Such case is not an error.

File: components/kyma-environment-broker/internal/storage/storage_test.go [hunk 2]
```diff
@@ -142,6 +144,11 @@ func TestSchemaInitializer(t *testing.T) {
 		err = svc.InsertProvisioningOperation(givenOperation)
 		require.NoError(t, err)
 
+		ops, err := svc.GetOperationsInProgressByType(dbsession.OperationTypeProvision)
+		require.NoError(t, err)
+		assert.Len(t, ops, 1)
+		assert.Equal(t, givenOperation.Operation, ops[0])
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/storage/storage_test.go]: there is an assertProvisioningOperation function

Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/storage/storage_test.go]: please, run the test and fix the build tag and makefile

Comment (polskikiel) [reply components/kyma-environment-broker/internal/storage/storage_test.go]: 👍

Comment (polskikiel) [reply components/kyma-environment-broker/internal/storage/storage_test.go]: about `assertProvisioningOperation`, I created a similar function `assertOperation`, which compares `internal.Operation` structs.

**Pull request merged**